pkgs/applications/audio/audacity/default.nix

```diff
@@ -166,16 +169,31 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
-    "-Daudacity_use_ffmpeg=linked"
+    "-DAUDACITY_REV_LONG=nixpkgs"
+    "-DAUDACITY_REV_TIME=nixpkgs"
     "-DDISABLE_DYNAMIC_LOADING_FFMPEG=ON"
+    "-Daudacity_conan_enabled=Off"
+    "-Daudacity_use_ffmpeg=loaded"
   ];
 
   doCheck = false; # Test fails
 
+  # Replace audacity's wrapper, to:
+  # - put it in the right place, it shouldn't be in "$out/audacity"
+  # - Add the ffmpeg dynamic dependency
+  postInstall = lib.optionalString stdenv.isLinux ''
+    rm "$out/audacity"
+    wrapProgram "$out/bin/audacity" \
+      --prefix LD_LIBRARY_PATH : "$out/lib/audacity":${lib.makeLibraryPath [ ffmpeg_4 ]} \
+      --suffix AUDACITY_MODULES_PATH : "$out/lib/audacity/modules" \
+      --suffix AUDACITY_PATH : "$out/share/audacity"
+  '';
+
   meta = with lib; {
     description = "Sound editor with graphical UI";
-    homepage = "https://www.audacityteam.org/";
-    license = licenses.gpl2Plus;
+    homepage = "https://www.audacityteam.org";
+    changelog = "https://github.com/audacity/audacity/releases";
+    license = with licenses; [ gpl2Plus gpl3 cc-by-30 ];
```

Comment:
Please replace `gpl3` with `gpl3Only` or `gpl3Plus`

Looking at https://github.com/audacity/audacity/discussions/2142 & they've been explicit about gplv2+ but not for 3 so I'd guess "only"

Reply:
@06kellyjac there they seem to say their binaries are gpl3 it seems? but this derivation builds its own binaries from source code. For GPLv3 in https://github.com/audacity/audacity/blob/master/LICENSE.txt they don't specify if only or plus, so it seems safer to just gpl3.

Reply:
https://www.audacityteam.org/about/license/ says gpl2 or later. i added a comment.

Reply:
I meant the about page in the application but since the quote mentions both the website and the application about page I'd guess they haven't had permission from all contributors yet or just haven't updated the website??

> Once we have permission from all contributors of active code, we will update the license text on the website and in Audacity's About dialog to say that the binaries are released under GPLv3.
https://github.com/audacity/audacity/discussions/2142